Formula/pari-galdata.rb

```diff
@@ -0,0 +1,21 @@
+class PariGaldata < Formula
+  desc "Galois resolvents data for PARI/GP"
+  homepage "https://pari.math.u-bordeaux.fr/packages.html"
+  url "https://pari.math.u-bordeaux.fr/pub/pari/packages/galdata.tgz"
+  # Refer to http://pari.math.u-bordeaux.fr/packages.html#packages for most recent package date
```

Comment:
@samford @nandahkrishna  could we parse from a string to a date (back to a string) in a livecheck block?

Reply:
It should be possible using a `strategy` block, we should ensure we extract the date next to the correct package name. I'll try coming up with a regex and strategy block for this in some time.

It might be easier to retrieve the version from [Debian](https://deb.debian.org/debian/pool/main/p/pari-galdata/) which has the version in the name of the source file, but I'm not sure if that would be preferable or up-to-date compared to the homepage.

Reply:
> It might be easier to retrieve the version from Debian which has the version in the name of the source file, but I'm not sure if that would be preferable or up-to-date compared to the homepage.

Yes, that's a concern (also why I didn't just use the Debian source packages instead). Most of the packages are old enough to be up-to-date but relatively recent updates (e.g. `pari-elldata`, updated in March 2021) still haven't been ported in.

If we were to try and retrieve the version from another arch, my vote would be for Arch Linux's [`PKGBUILD` manifests](https://github.com/archlinux/svntogit-community/blob/packages/pari-galdata/trunk/PKGBUILD), since they update more often.

Reply:
I'd say upstream is preferable. And otherwise Arch Linux because they're quick too.

Reply:
> could we parse from a string to a date (back to a string) in a livecheck block?

Yep, as Nanda mentioned, it's just a matter of creating a regex that captures the correct date and then we do the parsing in a `strategy` block. I've already created a couple `strategy` blocks that use `Date#parse` internally, so this isn't anything new.

> It might be easier to retrieve the version from [Debian](https://deb.debian.org/debian/pool/main/p/pari-galdata/) which has the version in the name of the source file, but I'm not sure if that would be preferable or up-to-date compared to the homepage.

Since the version information is available on the homepage (albeit in a format we have to parse), it would be much more preferable to check that.

---

I'll add some `livecheck` block suggestions in a moment, after I work this out (unless Nanda and beats me to it, ahah).